include/odp/api/spec/timer_types.h

```diff
@@ -477,9 +477,9 @@ typedef struct odp_timer_periodic_start_t {
 	/** First expiration time
 	 *
 	 *  The first expiration time in absolute timer ticks. When zero, the first expiration time
-	 *  is one period after the current time. After the first expiration, timer expiration
-	 *  continues with the defined frequency. The tick value must be within one timer period
-	 *  from the current time.
+	 *  is one period after the current time, or as close to that as the implementation can
+	 *  achieve. After the first expiration, timer expiration continues with the defined
+	 *  frequency. The tick value must be within one timer period from the current time.
```

Comment:
For the same reason, should "within" be changed to "less than" one period also here.

Reply:
I'm starting to remember why I used that word in the first place. So if it's "less than", then the first timeout can't be "one period after the current time". It would have to be something like "the first expiration time is one tick less than one period after the current time", or "the first expiration time is one period minus one tick after the current time".

Reply:
Or maybe it's fine. First expiration is "one period", but tick value is "less than".

In v2, changed to "less than" and also changed "from the current time" to "after the current time".